docs/userGuide/syntax/lists.md

```diff
@@ -72,6 +72,21 @@ Markdown can also be used in texts.
 You may need to add escape characters when using special characters for Markdown in text. 
 </box>
 
+**You can add an icon or text that is only for a specific item by using the `once` attribute.**
```

Comment:
Can you also add the fact that other attributes also won't get inherited? And add an example for this?